### PR TITLE
Added information about Linux Mint

### PR DESCRIPTION
**Pull request by akshay-theril**
Added linuxmint.md file ( shows information about Linux Mint distribution .)

### CODE REVIEW

File: linuxmint.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+>Impressive amount of customisation options
+
+>Good media support out of the box
+
+>Ideal for those switching from Windows/Mac
+
+## Basic Info
+
+* Based on : `debian`
+* Package Manager : `dpkg`
```
Comment (subins2000) [inline linuxmint.md]: give `apt`

File: linuxmint.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* Founded : ` 2006`
```
Comment (subins2000) [inline linuxmint.md]: an extra whitespace before `2006`. Please remove it.

File: linuxmint.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* Website : www.linuxmint.com
+
+## Download
+
+[Download Page](https://linuxmint.com/download.php)
+
+Releases are timed to be approximately one month after Ubuntu releases (which in turn are about one month after Gnome releases and two months after X Window System releases). Consequently, every Linux Mint release comes with an updated version of both GNOME and X and features some of the improvements brought in the latest Ubuntu release. Support for most releases is discontinued two months after the next release, but since the mid-2008 v5 every fourth release has been labelled a long-term support (LTS) version, indicating that it is supported (with updates) for longer, three years for v5 and v9, and five years thereafter.
```
Comment (subins2000) [inline linuxmint.md]: Make this paragraph less complicated and more friendly to newcomers.

File: linuxmint.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+>Impressive amount of customisation options
+
+>Good media support out of the box
+
+>Ideal for those switching from Windows/Mac
+
+## Basic Info
+
+* Based on : `debian`
+* Package Manager : `APT`
```
Comment (subins2000) [inline linuxmint.md]: make it lowercase

**Comment by subins2000** [comment]
I've merged the PR, but also make the changes I mentioned above and make another PR please !

**Comment by akshay-theril** [comment]
ok

**Pull request merged**